server/data/statementsClientTypes.ts

```diff
@@ -8,3 +8,18 @@ type StatementSummary = {
   isOverdue: boolean
   status: string
 }
+
+type Statement = {
+  id?: number
```

Comment:
I think some of these don't need to be optional?
including : id, bookingId, name, reporterName, incidentDate

Reply:
done